src/main/java/mate/academy/spring/dto/request/CinemaHallRequestDto.java

```diff
@@ -1,7 +1,12 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.NotNull;
+import org.hibernate.validator.constraints.Length;
+
 public class CinemaHallRequestDto {
+    @Length(min = 50, max = 150)
```

Comment:
From common mistakes:
For validation use classes from javax.validation package.
Be careful with @Size, @Min and @Max annotations. Read javadoc to learn about their correct use.